ethcore/src/miner/transaction_queue.rs

```diff
@@ -355,11 +357,11 @@ impl Default for TransactionQueue {
 impl TransactionQueue {
 	/// Creates new instance of this Queue
 	pub fn new() -> Self {
-		Self::with_limit(1024)
+		Self::with_limits(1024, !U256::zero())
 	}
 
 	/// Create new instance of this Queue with specified limits
-	pub fn with_limit(limit: usize) -> Self {
+	pub fn with_limits(limit: usize, tx_gas_limit: U256) -> Self {
```

Comment:
It's changed by setter anyway, other stuff like `minimal_gas_price,gas_limit,etc` is not here also (because there are sensible defaults / it's unbounded without negative consequences)